agent/wsclient/client.go

```diff
@@ -161,14 +182,22 @@ func (cs *ClientServerImpl) Connect() error {
 		seelog.Warnf("Error creating a websocket client: %v", err)
 		return fmt.Errorf(string(resp) + ", " + err.Error())
 	}
-	cs.Conn = websocketConn
+	cs.conn = websocketConn
 	return nil
 }
 
+func (cs *ClientServerImpl) IsConnected() bool {
+	return cs.conn != nil
```

Comment:
I'm using this function to replace nil checks elsewhere in the agent code. I might need to change the name if it implies the wrong state. Does `IsReady()` convey that idea better?